client/my-sites/stats/controller.jsx

```diff
@@ -446,6 +446,9 @@ export default {
 		if ( siteId && ! isJetpack ) {
 			page.redirect( '/stats' );
 		} else {
+			// FIXME: Auto-converted from the Flux setTitle action. Please use <DocumentHead> instead.
+			context.store.dispatch( setTitle( i18n.translate( 'Stats', { textOnly: true } ) ) );
```

Comment:
How about using `<DocumentHead>` instead? See the [comments section](https://github.com/Automattic/wp-calypso/blob/37b89f4cc9e1514a4186a2796752831ce987766c/client/my-sites/comments/main.jsx#L82) for an example.

Do you have time? If not, I can make this change for you. I'd just prefer we don't willingly introduce obviated code, especially when it's easy not to.

Reply:
I actually am probably not going to get to doing this properly, so if you can handle it, I'd really appreciate it @dmsnell, thanks. I'll keep an eye out on here and be happy to test though!

Reply:
Just a note that a switch to `DocumentHead` is causing some issues with the title here (try switching to `Weeks` for example). On the other hand, the good ol' obviated code is working as expected. :) I logged an issue https://github.com/Automattic/wp-calypso/issues/23924 for it, hopefully we'll be able to get to it soon.